### ISSUE TITLE
[Suggestion]: Randomized configs requests

### ISSUE THREAD

**Issue by Composer-Hu** [nonamecrackers2/crackers-wither-storm-mod#1754]
### Description of feature request

Add a random configuration item like random_stroll to the existing fixed configuration such as tractor_beam_pull_speed,roaring,cluster_size (I'm not sure if my analogy is appropriate), but it means that everything is completely random and changes at any time. Or even set a random range of values,  for example, clusters size: 0-4 (I think this might be asking too much). Thank you.

**Comment by WitherBean** [comment]
The cluster size would be good, but a random tractor beam speed pull would just not look as good.

**Comment by Composer-Hu** [comment]
> The cluster size would be good, but a random tractor beam speed pull would just not look as good.

So my idea is to add a random possibility to a fixed number, it doesn't affect anything that's already there.

**Comment by TaintedBlockmObile** [comment]
I kinda like this idea

**Comment by TaintedBlockmObile** [comment]
> The cluster size would be good, but a random tractor beam speed pull would just not look as good.

Depends on how it's implemented tbh

**Comment by Composer-Hu** [comment]
> > The cluster size would be good, but a random tractor beam speed pull would just not look as good.
> 
> Depends on how it's implemented tbh

Yes, I don't know how to express "random", after all, it is an abstract concept, right

**Comment by WitherBean** [comment]
> > > The cluster size would be good, but a random tractor beam speed pull would just not look as good.
> > 
> > 
> > Depends on how it's implemented tbh
> 
> Yes, I don't know how to express "random", after all, it is an abstract concept, right

Do you mean each phase has a chance to select a pull speed?

**Comment by Composer-Hu** [comment]
> > > > The cluster size would be good, but a random tractor beam speed pull would just not look as good.
> > > 
> > > 
> > > Depends on how it's implemented tbh
> > 
> > 
> > Yes, I don't know how to express "random", after all, it is an abstract concept, right
> 
> Do you mean each phase has a chance to select a pull speed?
No, I mean every target.

**Comment by Composer-Hu** [comment]
> > > > The cluster size would be good, but a random tractor beam speed pull would just not look as good.
> > > 
> > > 
> > > Depends on how it's implemented tbh
> > 
> > 
> > Yes, I don't know how to express "random", after all, it is an abstract concept, right
> 
> Do you mean each phase has a chance to select a pull speed?

No, I mean every target.

**Comment by Composer-Hu** [comment]
I just think of a good example:
/witherstormmod config set server.roaring.maximumRoarInterval 16
/witherstormmod config set server.roaring.minimumRoarInterval 4
Maybe all configs that can write their own values could be like this one, and when two values are the same, it's the same value forever.

**Comment by NinjaSam5399** [comment]
I would love ranged config.

**Comment by BluePineappleGuy** [comment]
Here's an idea as to how it could work: 
Lets say that 'Tractor beam pull speed' is set to 0.2. If I change it to 0.6 it will always pull at that speed, but saying '0.2-0.4' would allow it to go anywhere between that.
A separate config could exist to specify the chances a little more (i.e. its ultimate target is more likely to receive smaller values, whereas side targets are more likely to receive bigger ones).
